### PR TITLE
Add invoice endpoints

### PR DESCRIPTION
**Pull request by allcaps**

### CODE REVIEW

File: tests/test_invoices.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+def test_get_invoice(client, response):
+    """Retrieve a single invoice."""
+    response.get('https://api.mollie.com/v2/invoices/%s' % INVOICE_ID, 'invoice_single')
+
+    invoice = client.invoices.get(INVOICE_ID)
+    assert isinstance(invoice, Invoice)
+    assert invoice.id == INVOICE_ID
+    assert invoice.resource == 'invoice'
```
Comment (whyscream) [inline tests/test_invoices.py]: When testing the object properties, test all the properties. Coverage is your friend here: https://travis-ci.org/mollie/mollie-api-python/jobs/535306816

File: mollie/api/objects/invoice.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    @property
+    def gross_amount(self):
+        return self._get_property('grossAmount')
+
+    @property
+    def lines(self):
+        return self._get_property('lines') or []
+
+    @property
+    def links(self):
```
[the remark this answers is not in the document]
Comment (allcaps) [reply mollie/api/objects/invoice.py]: Done

File: tests/responses/invoice_single.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+             "href": "https://api.mollie.com/v2/invoices/inv_xBEbP9rvAq",
+             "type": "application/hal+json"
+        },
+        "pdf": {
+             "href": "https://www.mollie.com/merchant/download/invoice/xBEbP9rvAq/2ab44d60b35b1d06090bba955fa2c602",
+             "type": "application/pdf",
+             "expiresAt": "2018-11-09T14:10:36+00:00"
+        }
+    }
+}
```
[the remark this answers is not in the document]
Comment (allcaps) [reply tests/responses/invoice_single.json]: Done

File: tests/responses/invoice_single.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+                "currency": "EUR"
+            }
+        }
+    ],
+    "_links": {
+        "self": {
+             "href": "https://api.mollie.com/v2/invoices/inv_xBEbP9rvAq",
+             "type": "application/hal+json"
+        },
+        "pdf": {
```
[the remark this answers is not in the document]
Comment (allcaps) [reply tests/responses/invoice_single.json]: Done

**Pull request merged**